tests/e2e_tests.py

```diff
@@ -33,6 +33,17 @@ def test_print_only_no_link(link_localtime_mock, deb_tz_mock):
     assert_false(deb_tz_mock.called)
 
 
+@httpretty.activate
+@mock.patch("tzupdate.write_debian_timezone")
+@mock.patch("tzupdate.link_localtime")
+def test_print_sys_tz_no_link(link_localtime_mock, deb_tz_mock):
+    setup_basic_api_response()
```

Comment:
Huh? you don't call any http methods here...

In general, this increases coverage but doesn't actually test anything. Please do check `get_sys_timezone` with a unit test :-)